### ISSUE TITLE
Tailoring the load.py for using my own data

### ISSUE THREAD

**Issue by kapilagrawal95** [clinicalml/dmm#2]
Hi
I have a time series data almost similar to jsb_chorales data. The only difference is that for each timestep jsb_chorales has 88 binary values while I have one continuous value so essentially my one sequence can be represented in a 1-d array. I would like to train a dmm on my data.

I tried to understand the simulateLinearData but I couldn't entirely comprehend the code; mainly because of this line:
zlist= [np.copy(z)[:,None,:]]
I am unable to visualize it.

**Comment by rahulk90** [comment]
I could be misunderstanding your question but you don't need to use simulateLinearData for setting up the dataset. Yo have to format your data into a 3D tensor with shape: `N x T x 1` (in your case since you have one-dimensional data)

You may find this ipython notebook helpful for setting up the data (See `A] Data`): 
[Notebook - Setup](https://github.com/clinicalml/dmm/blob/3a2e8cec5e58c5a141a6da6e685378e269e69bb8/ipynb/DMM-Setup.ipynb)

**Comment by kapilagrawal95** [comment]
Thanks! That resolved my above query. 
Another thing: my dataset's time-series vary. Will the algorithm work in that case? And will the 3-d tensor have shape:  N x (T_max) x 1?

**Comment by rahulk90** [comment]
Yes. Search for `mask` in the ipython notebook. It should tell you how to specify variable length sequences.
